RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -147,7 +147,9 @@
 eras.trackingPhase2PU140.toReplaceWith(detachedQuadStepTrajectoryFilterBase,
     _detachedQuadStepTrajectoryFilterBase.clone(
```

Comment:
Yes.

Also for the Chi2Estimators, don't remove parameters `clusterChargeCut` and `pTChargeCutThreshold`, but set `clusterChargeCut = dict(refToPSet_ = 'SiStripClusterChargeCutNone')` (look how `trackingPhase1PU70` is configured). I know at edmConfigDump-level things look a bit different, but `Chi2ChargeMeasurementEstimator` with `SiStripClusterChargeCutNone` is functionally equivalent to `Chi2MeasurementEstimator`.